Week2/homework/project-grade-calculator.js

```diff
@@ -0,0 +1,40 @@
+
+// PROJECT: Grade calculator
+//a script that calculates grades, based on the American grading system!
+
+{
+    'use strict'
+
+    let result;
+    //defined a maximum score. It can be changed according to test.
+    let maxScore = 100;
+
+    //turns scores into percentage.
+    result = ((result / maxScore) * 100);
+
+    //checking the score's equivalent of usa grades. 
+    function gradeCalculator(result) {
+        let grade;
+
+        if (result >= 0 && result <= 49) {
+            grade = 'F';
+        } else if (result >= 50 && result <= 59) {
+            grade = 'E';
+        } else if (result >= 60 && result <= 69) {
+            grade = 'D';
+        } else if (result >= 70 && result <= 79) {
+            grade = 'C';
+        } else if (result >= 80 && result <= 89) {
+            grade = 'B';
+        } else if (result >= 90 && result <= 100) {
+            grade = 'A';
```

Comment:
I think you don't necessarily need to have the first part of the else-if conditions here. It's good you are being explicit but your code would also work fine without them. Also, I would suggest adding another condition at the top to check if grade is a number and falls between 0 and 100 🙂